docs/troubleshooting.md

```diff
@@ -142,3 +142,11 @@ Based on the above log, you can add the following entry to remove the operation:
 ```json
 {"op":"remove", "path":"/spec/template/spec/hostNetwork"}
 ```
+
+### My bundle which has a Horizontal Pod Autoscaler is in state `Modified`
+
+For bundles with an HPA is expected that it is in state `Modified`, as the bundle contains fields that differ from the state of the Bundle at deployment.
+
+In such cases you have to define a patch in the `fleet.yaml` to ignore this field according to [bundle-diffs](./bundle-diffs.md).
```

Comment:
```suggestion
You must define a patch in the `fleet.yaml` to ignore this field according to [bundle-diffs](./bundle-diffs.md).
```

Reply:
Please specify what field you're referring to when you say `ignore this field`. Ty!